### ISSUE TITLE
Store filtering settings

### ISSUE THREAD

**Issue by tzarebczan** [OdyseeTeam/odysee-frontend#2640]
use case: a user filters out shorts via the duration setting on their following page. If you leave and come back, it disappears. 

We can either store by default, per page, or make it a setting in filter to persist (never, on current page, on all pages)

**Comment by infinite-persistence** [comment]
Some notes to pass along:
- Most of the settings are currently already saved "per page" via local storage, e.g. `usePersistedState('minDurUserMinutes-${location.pathname}'`
- "Duration" diverged/stopped doing that because the LS is being clogged up with these items from each page visit.  LS size is limited, seeing quota errors.
- The last suggestion sound great, but maybe do it at Redux (similar to watch history) to avoid quota issues?

**Comment by tzarebczan** [comment]
Yep, recall those issues with size. Fine with doing it in redux...this is not super high priority either

**Comment by mcgravier** [comment]
Shorts spam coming from Youtube is getting worse every day, degrading the user experience. Please increase the priority of this feature - ability to filter out shorts is slowly becoming a necessity for comfortable content consumption
